doozerlib/rpmcfg.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 import glob
-import io
```

Comment:
think ya still need this

Reply:
`io` is still in there. I replaced a dupe import.